test/common/alerts_exist.go

```diff
@@ -481,6 +481,63 @@ var commonExpectedRules = []alertsTestRule{
 			"SendgridSmtpSecretExists",
 		},
 	},
+	{
+		File: "redhat-rhmi-3scale-zync-que.yaml",
```

Comment:
Could we remove the hard-coded namespace prefix? Depending on the install type, it could be `redhat-rhmi/redhat-rhoam`, or whatever the desired prefix is.

Reply:
Oh yeah, good point, will do, thanks Rob :+1: